src/antsibull_changelog/plugins.py

```diff
@@ -582,10 +590,18 @@ def load_plugins(
         )
         store_yaml(plugin_cache_path, plugins_data)
 
-    plugins = PluginDescription.from_dict(plugins_data["plugins"])
+    plugins = PluginDescription.from_dict(
+        plugins_data["plugins"],
+        category="plugins",
```

Comment:
```suggestion
```
This one wasn't there before :-)

Reply:
> Seems I've 'fixed it to death' and the changelog has no modules, but I'm not sure where my error is.

This is the one ;)

Reply:
heh copy pasta error!  TIL how to check my diffs better in vscode ;-)